test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -137,5 +138,52 @@ public static void RemoveSetting(XDocument doc, string key)
                 item.Remove();
             }
         }
+
+        // Add NetStandard.Library and NetCorePlatforms to the feed and save the file.
+        public void AddNetStandardFeeds()
+        {
+            var reposRoot = GetRepositoryRootDirectory();
+            var netStandardLibraryPackageFeed = GetNetStandardLibraryFeed(reposRoot);
+            var netCorePlatformsPackageFeed = GetNetCorePlatformFeed(reposRoot);
+
+            Assert.True(Directory.Exists(netStandardLibraryPackageFeed));
+            Assert.True(Directory.Exists(netCorePlatformsPackageFeed));
+
+            var section = GetOrAddSection(XML, "packageSources");
+            AddEntry(section, "NetStandardLibrary", netStandardLibraryPackageFeed);
+            AddEntry(section, "NetCorePlatforms", netCorePlatformsPackageFeed);
```

Comment:
These two lines are adding the first level of a v3 folder as a package source to the nuget.config.

Firstly, I don't understand why this works because the nupkg will be in the relative path `2.0.0/netstandardlibrary.2.0.0.nupkg` within the source. It's not a flat folder of nupkgs, and it's not a valid v2 folder structure. Rhetorical question: how/why does NuGet find it? 

Secondly, Can you just add the `<repo root>/packages` folder as a source, and let it find both packages from the one source, rather than having two sources, each with a single package?

Reply:
> It's not a flat folder of nupkgs, and it's not a valid v2 folder structure.

It's really looking at *.nupkg, ignoring the existence of other files/folders :)

Reply:
Thanks for @nkolev92 help answer the first question. 
For the second, since there're 21 packages under `<repo root>/packages`, I think only adding the two packages we need is more accurate than adding all of them.
E.g. if we have a test: creating a netstandard2.0 project, then install newtonsoft.json without adding feed for that. We expect this test to fail as no feed added for package newtonsoft.json.
But if we add  `<repo root>/packages` as the feed, the test will pass incorrectly.